.release-please-manifest.json

```diff
@@ -1 +1 @@
-{"fxgcppubsub":"1.6.0","fxslack":"1.0.0","fxredis":"1.1.0","fxgomysqlserver":"1.0.0","fxjsonapi":"1.0.0"}
\ No newline at end of file
+{"fxgcppubsub":"1.6.0","fxslack":"1.0.0","fxredis":"1.1.0","fxgomysqlserver":"1.0.0","fxjsonapi":"1.0.0","fxelasticsearch":"1.0.0"}
```

Comment:
That should be reverted, the bot handle this file itself 👍

Reply:
I meant you should revert the full update of this file, the bot will update it itself